.github/workflows/wheels.yml

```diff
@@ -71,7 +83,7 @@ jobs:
       - name: Build wheel
         shell: bash
         run: |
-            python3 -mpip install wheel
+            python3 -mpip install wheel setuptools
```

Comment:
I added this to fix this failure: https://github.com/pytorch/tensordict/actions/runs/9555748210/job/26354755267?pr=818

In Python 3.12, setuptools is required to be able to import distutils

Not sure if this is really the best place to add this dependency

Reply:
I think it's ok, long term we should use smth like what torchtune has (ie no setup.py at all)
https://github.com/pytorch/torchtune/blob/main/pyproject.toml